src/main/java/suppliers/ColorSupplier.java

```diff
@@ -0,0 +1,11 @@
+package suppliers;
+
+import core.basesyntax.Colors;
+import java.util.Random;
+
+public class ColorSupplier {
+    public String getRandomColor() {
+        int index = new Random().nextInt(Colors.values().length);
```

Comment:
lets make Random as class level variable, not to create new instance of Random class every time we call getRandomColor() method